### DISCUSSION TITLE
Feriado

### DISCUSSION THREAD

**Discussion by marceloferrer** [itAtPPI/ppi-connector-python#1] [Ideas]
Estaría bueno si podrían aplicar una lista de feriados financieros para que al realizar un programa, tenga en cuenta esas fechas al momento de intentar operar.

**Comment by itAtPPI** [comment]
Buenos días, en la próxima actualización tenemos pensado implementar 3 nuevos endpoints para la gestión de feriados. El primero que permita la búsqueda de feriados entre 2 fechas, un segundo que al consumirlo devuelva si el día actual es feriado local o no y un tercero que permitirá saber si el día actual es feriado en estados unidos.

**Comment by itAtPPI** [comment]
Buenos días, en la nueva versión de la api que se liberó en el día de hoy se encuentran los nuevos endpoints mencionados en el comentario anterior. Para utilizarlos podes llamarlos de la siguiente forma desde python:

        # Get holidays
        print("\nGet local holidays for the current year")
        holidays = ppi.configuration.get_holidays(start_date=datetime(2022, 1, 1), end_date=datetime(2022, 12, 31))
        for holiday in holidays:
            print("%s - %s " % (holiday["date"][0:10], holiday["description"]))
        
        print("\nGet USA holidays for the current year")
        holidays = ppi.configuration.get_holidays(start_date=datetime(2022, 1, 1), end_date=datetime(2022, 12, 31),
                                                  is_usa=True)
        for holiday in holidays:
            print("%s - %s " % (holiday["date"][0:10], holiday["description"]))

        # Check holidays
        print("\nIs today a local holiday?")
        print(ppi.configuration.is_local_holiday())
        print("\nIs today a holiday in the USA?")
        print(ppi.configuration.is_usa_holiday())
